docs/api/index.rst

```diff
@@ -11,12 +11,18 @@ API
 Changelog
 ---------
 
+1.11 (unreleased)
+'''''''''''''''''
+
+- Parent metadata is now readable if children creation is allowed
+- Now returns a ``412`` instead of a ``403`` if the ``If-None-Match: *`` header is
+  provided and the ``create`` permission is allowed
```

Comment:
Do you want to add a note about the fact that `GET`ing the list of records will return an empty list (and not a 403) if the user doesn't have the read permission on the collection, and has not created any records yet? (as discussed in the issue #803)